scv-prototype-frontend/src/app/views/Login/Login.jsx

```diff
@@ -15,9 +16,9 @@ const Login = (props) => {
 	const [password, setPassword] = useState('password');
 	const [rememberMe, setRememberMe] = useState();
 
-	const {t} = useTranslation();
-	const {error, loading, login} = useApi();
-	const {authenticationContext} = {...useContext(AuthenticationContext)};
+	const { t } = useTranslation();
+	const { error, loading, login } = useApi();
+	const { authenticationContext } = { ...useContext(AuthenticationContext) };
```

Comment:
Are we using `{ foo }` or `{foo}` here. The react style guide doesn't specifically mention decomposition; it mentions using `{...}` _in react props_.

Reply:
The spacings were added by VS Code when I saved the file.